suite/tests/CMakeLists.txt

```diff
@@ -2090,6 +2090,18 @@ if (DR_HOST_AARCH64)
     # with some tool chains (such as SUSE GCC 6.2) results in an executable that
     # is not properly static. Adding --no-export-dynamic avoids the problem.
     "-Wl,--no-export-dynamic")
+
+  add_exe(allasm_aarch64_flush
+    ${PROJECT_SOURCE_DIR}/clients/drcachesim/tests/allasm_aarch64_flush.asm
+    "-early_inject" "")
```

Comment:
early_inject is the default so no need to have it here (and we might drop support for late)

Reply:
Done.